modules/tinymce/src/core/main/ts/bookmark/ResolveBookmark.ts

```diff
@@ -15,6 +15,14 @@ import {
 } from './BookmarkTypes';
 import * as CaretBookmark from './CaretBookmark';
 
+export interface BookmarkResolveResult {
+  range: Range;
+  forward: boolean;
```

Comment:
```suggestion
  readonly range: Range;
  readonly forward: boolean;
```

Reply:
I guess we could try to setup some form of waiter that waits for the visual chars wrapping spans to appear. I avoided adding a test since it's throttled and needs to wait a while until we can check if the selection is still backwards. But I see now that it can be a wait for a specific condition. I usually hate the other kind where it needs to just wait a while for something to not happen since it's likely to flake or pass when it shouldn't pass so if I can avoid that I will add a test if I can't then I will not add a test for the original case. .